src/console/mirrorManager.ts

```diff
@@ -193,7 +193,7 @@ export class MirrorManager extends EventEmitter {
 
     details.connection.disconnect();
     if (details.autoSwitcher) {
-      details.autoSwitcher.disconnect();
+      void details.autoSwitcher.disconnect();
```

Comment:
Can we just make the disconnect function an async function and then use await here? We really should avoid using `void` unless we have to like in a callback or something.